src/build/ci/find-unapproved-public.sh

```diff
@@ -24,9 +24,20 @@ count=$(grep -E "public.*(class|interface|enum|record)" src/main/java/org/apache
   grep -v " class ParsedAccessExpression " |
   grep -v " enum ExpressionType " |
   grep -v " class Authorizations " |
+  grep -v " class Bytes " |
   grep -c -v " class InvalidAccessExpressionException ")
 
+files=$(grep -E "public.*(class|interface|enum|record)" src/main/java/org/apache/accumulo/access/*.java |
```

Comment:
There is probably an easier way to get the count of the `files` array, but I didn't look that hard at the `bash` man page.